### PR TITLE
Create non-integer-values.js

### PR DESCRIPTION
**Pull request by Yaffle**
Try in Google Chrome

### CODE REVIEW

File: test/built-ins/Date/UTC/non-integer-values.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
```
Comment (littledan) [inline test/built-ins/Date/UTC/non-integer-values.js]: Who was this test written by?

Comment (Yaffle) [reply test/built-ins/Date/UTC/non-integer-values.js]: oh, sorry, forgot to change this line

Comment (littledan) [reply test/built-ins/Date/UTC/non-integer-values.js]: Could you make a follow-up PR to fix this line?

Comment (Yaffle) [reply test/built-ins/Date/UTC/non-integer-values.js]: @littledan , https://github.com/tc39/test262/pull/1393

**Pull request merged**